server/graphql-schema.js

```diff
@@ -620,6 +620,14 @@ const graphqlSchema = gql`
         """
         See Assertion for more information.
         """
+        assertion: AssertionResultInput
```

Comment:
@alflennik we added an assertion and priority to the schema because the [getMetrics function](https://github.com/w3c/aria-at-app/blob/main/server/util/getMetrics.js) uses the priority to do calculations

Reply:
I think one way to load that data would be to use the testResultsResolver in server/resolvers/TestPlanRun/testResultsResolver.js which will populate that association.